gcloud-java-pubsub/src/main/java/com/google/cloud/pubsub/spi/v1/PublisherApi.java

```diff
@@ -402,27 +347,28 @@ public final PublishResponse publish(String topic, List<PubsubMessage> messages)
    * }
    * </code></pre>
    *
-   * <!-- manual edit -->
-   * <!-- end manual edit -->
-   *
    * @param request The request object containing all of the parameters for the API call.
    * @throws com.google.api.gax.grpc.ApiException if the remote call fails
    */
-  public PublishResponse publish(PublishRequest request) {
+  public final PublishResponse publish(PublishRequest request) {
     return publishCallable().call(request);
   }
 
-  // AUTO-GENERATED DOCUMENTATION AND METHOD - see instructions at the top of the file for editing.
+  // AUTO-GENERATED DOCUMENTATION AND METHOD
   /**
-   * Adds one or more messages to the topic. Generates `NOT_FOUND` if the topic
+   * Adds one or more messages to the topic. Returns `NOT_FOUND` if the topic
```

Comment:
I filed an issue to make sure we don't lose track: https://github.com/googleapis/googleapis/issues/87